src/index.js

```diff
@@ -11,8 +11,9 @@ const _createFabricRefCode = (refIdentifier, typeIdentifier, propsIdentifier) =>
     'dataElement',
     'dataComponent',
     'dataSourceFile',
-  ];  
-  if (global.__turboModuleProxy != null && Platform.OS === 'ios') {
+  ]; 
+  const isTurboModuleEnabled = global.RN$Bridgeless || global.__turboModuleProxy != null
```

Comment:
is this check safe for older RN versions? Also, the $ character is allowed as a field name in js?

Reply:
`global.RN$Bridgeless` will just return `undefined` for older versions and yes, `$` is safe